nixos/modules/services/web-apps/invidious.nix

```diff
@@ -155,6 +159,27 @@ in
       description = lib.mdDoc "The Invidious package to use.";
     };
 
+    automaticRestart = {
```

Comment:
Do we need these options at all? Is there ever a reason why somebody would not want the auto-restarting behavior or customize its timing? Because if not we don't need to introduce these options

Reply:
I genuinely don't know. I don't mind having no option. If a user wants to disable it, he can always manually set `systemd.services.invidious.serviceConfig.runtimeMaxSec = "0"`.

However I don't feel like adding those options is a problem.

-> Both solutions are fine to me.

Reply:
@infinisil what is your opinion on this ? Should we leave the option (for me, it doesn't hurt anyone) or simply set the restart policy directly (with `mkDefault` though) ?

Reply:
I'd say let's not have these options, they really feel like an implementation detail that doesn't need to be exposed to users. And yeah users can still customize the underlying options if really needed.

Reply:
Ok done.